CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# Introduction
+
+VM Import operator requires the following components to be installed:
+
+* **Kubernetes** is a container orchestration system, and is used to run
```

Comment:
When using k8s we would not able able to install templates. I suggest to change it to OKD.

Reply:
I added an explanation below - see line 16:
```
VM Import operator relies on templates to create best-match VM specification.
However, templates aren't k8s nor kubevirt resources, but are part of OpenShift API,
therefore as a base provider for development should be OKD / OpenShift to allow
enhancements of templates.
```

Reply:
+added a note at the same bullet